tests/bin/keys.test.ts

```diff
@@ -212,7 +230,7 @@ describe('CLI keys', () => {
         });
         await polykeyAgent.keyManager.changeRootKeyPassword(password);
       },
-      global.defaultTimeout * 2,
+      global.polykeyStartupTimeout * 4,
```

Comment:
Ever increasing timeouts are not a solution. We need to improve our tests or get rid of them.